templates/edit.html

```diff
@@ -56,10 +56,15 @@
 			<button>Submit</button>
 		</form>
 		{% if is_admin %}
-			<button onclick="delete()">Delete Entry</button>
+			<form onsubmit="del()" method="post" action="/delete?id={{ content['_id'] }}">
+				<button>Delete Entry</button>
+			</form>
 			<script>
-				function delete() {
-					alert("this does not work yet")
+				function del() {
+					if(!confirm("Are you sure you want to delete "+window.location.pathname)) {
```

Comment:
It should be using the actual entry title from Jinja2 instead of window.location.pathname, since the url can be the id sometimes